src/cli/utils/sharepoint.go

```diff
@@ -82,22 +82,19 @@ func MakeSharePointOpts(cmd *cobra.Command) SharePointOpts {
 func SharePointAllowedCategories() map[string]struct{} {
 	return map[string]struct{}{
 		flags.DataLibraries: {},
-		// flags.DataLists:     {}, [TODO]: uncomment when lists are enabled
```

Comment:
same here, please keep changes separate.  one PR for releasing support, one PR for testing it.